site/content/docs/commands/task-run.md

```diff
@@ -17,7 +17,7 @@ Generally, the steps involved in task run are:
     1. Tasks with the same group name share the same set of resources, including the CloudFormation stack, ECR repository, CloudWatch log group and task definition.
     2. If the tasks are deployed to a Copilot environment (i.e. by specifying `--env`), only public subnets that are created by that environment will be used. 
     3. The `--env` flag only works with environments created with v0.3.0 of Copilot or later. Customers using environments created with v0.2.0 or earlier can update their environment manager role with [this](https://github.com/aws/copilot-cli/blob/mainline/templates/environment/cf/environment-manager-role.yml) policy. 
-    4. If you are using the `--default` flag and get an error saying there's no default cluster, run `aws ecs create-cluster` and then re-run the Copilot command. 
+    4. If you doesn't use the `--env` flag Copilot run task on default cluster in your region so when you get an error saying there's no default cluster, run `aws ecs create-cluster` and then re-run the Copilot command. 
```

Comment:
- Would you mind removing 3. above? now that the CLI released version 1.x I think we can remove it :)
- Maybe instead of having this in our documentation, can we instead move it to the code? That will provide more immediate feedback to users:
https://github.com/aws/copilot-cli/blob/e309bef2866eb1e0eeb36ab0bf947248839a07f7/internal/pkg/cli/task_run.go#L406-L408

```go
if !hasDefaultCluster {
	log.Errorf(`Looks like there is no "default" cluster in your region!
Please run %s to create the cluster first, and then re-run %s.
`, color.HighlightCode("aws ecs create-cluster"), color.HighlightCode("copilot task run"))
	return errors.New(`cannot find a "default" cluster to deploy the task to`)
}
```